diff -up transcode-1.1.7/filter/filter_resample.c.oldabi transcode-1.1.7/filter/filter_resample.c
--- transcode-1.1.7/filter/filter_resample.c.oldabi	2011-11-19 17:50:27.000000000 +0100
+++ transcode-1.1.7/filter/filter_resample.c	2012-05-26 11:16:46.986521686 +0200
@@ -112,7 +112,7 @@ static int resample_configure(TCModuleIn
 
     pd->resample_ctx = av_audio_resample_init(vob->a_chan, vob->a_chan,
                                            vob->mp3frequency, vob->a_rate,
-                                           AV_SAMPLE_FMT_S16, AV_SAMPLE_FMT_S16,
+                                           SAMPLE_FMT_S16, SAMPLE_FMT_S16,
                                            16, 10, 0, 0.8);
     if (pd->resample_ctx == NULL) {
         tc_log_error(MOD_NAME, "can't get a resample context");
