--- transcode-1.0.0/pvm3/Makefile.in.pvmbin	2005-07-12 10:40:55.000000000 +0300
+++ transcode-1.0.0/pvm3/Makefile.in	2005-07-21 19:41:14.000000000 +0300
@@ -763,11 +763,6 @@
 	uninstall-info-am uninstall-pkgLTLIBRARIES
 
 
-install-data-local:
-	@$(NORMAL_INSTALL)
-	if test ! -e $(bindir)/pvmgs; then \
-		ln -s $(PVM3_PVMGS) $(bindir)/pvmgs; \
-	fi;
 # Tell versions [3.59,3.63) of GNU make to not export all variables.
 # Otherwise a system limit (for SysV at least) may be exceeded.
 .NOEXPORT:
--- transcode-1.0.0/pvm3/Makefile.am.pvmbin	2005-07-12 10:25:16.000000000 +0300
+++ transcode-1.0.0/pvm3/Makefile.am	2005-07-21 19:41:35.000000000 +0300
@@ -43,8 +43,3 @@
 	pvm_version.h \
 	vob_pack_unpack.h
 
-install-data-local:
-	@$(NORMAL_INSTALL)
-	if test ! -e $(bindir)/pvmgs; then \
-		ln -s $(PVM3_PVMGS) $(bindir)/pvmgs; \
-	fi;
