diff -up transcode-1.1.5/export/export_ogg.c.no_video transcode-1.1.5/export/export_ogg.c
--- transcode-1.1.5/export/export_ogg.c.no_video	2010-01-30 19:05:38.000000000 +0200
+++ transcode-1.1.5/export/export_ogg.c	2010-01-30 19:05:59.000000000 +0200
@@ -89,8 +89,9 @@ MOD_open
 	    return(TC_EXPORT_ERROR);
 	}
 
-	if (!strcmp(vob->video_out_file, vob->audio_out_file)) {
-	    tc_log_info(MOD_NAME, "Writing audio to \"/dev/null\" (no -m option)");
+	if (vob->video_out_file && 
+	    !strcmp(vob->video_out_file, vob->audio_out_file)) {
+	  tc_log_info(MOD_NAME, "Writing audio to \"/dev/null\" (no -m option)");
 	}
 	if (vob->mp3bitrate == 0)
 	    result = tc_snprintf (buf, PATH_MAX, "oggenc -r -B %d -C %d -q %.2f %s -Q -o \"%s\" %s -",
@@ -214,10 +215,11 @@ MOD_close
 	pFile = NULL;
 
 	if (verbose > 0 && strcmp (vob->audio_out_file, "/dev/null") &&
-		strcmp (vob->video_out_file, "/dev/null")!=0) {
-	    tc_log_info (MOD_NAME, "Hint: Now merge the files with");
-	    tc_log_info (MOD_NAME, "Hint: ogmmerge -o complete.ogg %s %s",
-			 vob->video_out_file, vob->audio_out_file );
+	    vob->video_out_file &&
+	    strcmp (vob->video_out_file, "/dev/null")!=0) {
+	  tc_log_info (MOD_NAME, "Hint: Now merge the files with");
+	  tc_log_info (MOD_NAME, "Hint: ogmmerge -o complete.ogg %s %s",
+		       vob->video_out_file, vob->audio_out_file );
 	}
 
         return(0);
