diff -up transcode-1.1.6/import/probe_v4l.c.header transcode-1.1.6/import/probe_v4l.c
--- transcode-1.1.6/import/probe_v4l.c.header	2011-11-06 09:35:02.000000000 +0100
+++ transcode-1.1.6/import/probe_v4l.c	2011-11-09 22:58:47.082990096 +0100
@@ -31,7 +31,7 @@
 
 #include <sys/ioctl.h>
 
-#include "v4l/videodev.h"
+#include <libv4l1-videodev.h>
 
 #if defined(HAVE_LINUX_VIDEODEV2_H) && defined(HAVE_STRUCT_V4L2_BUFFER)
 #define _LINUX_TIME_H
diff -up transcode-1.1.6/import/v4l/import_v4l.c.header transcode-1.1.6/import/v4l/import_v4l.c
--- transcode-1.1.6/import/v4l/import_v4l.c.header	2011-11-06 09:35:02.000000000 +0100
+++ transcode-1.1.6/import/v4l/import_v4l.c	2011-11-09 22:21:37.225106789 +0100
@@ -29,7 +29,7 @@
 #include <sys/ioctl.h>
 #include <sys/mman.h>
 
-#include "videodev.h"
+#include <libv4l1-videodev.h>
 
 #define MOD_NAME    "import_v4l.so"
 #define MOD_VERSION "v0.2.0 (2008-10-26)"
