diff -up transcode-1.0.4/configure.in.ffmpeg transcode-1.0.4/configure.in
--- transcode-1.0.4/configure.in.ffmpeg	2008-08-12 19:53:21.000000000 +0300
+++ transcode-1.0.4/configure.in	2008-08-12 19:57:30.000000000 +0300
@@ -611,14 +611,14 @@ AC_ARG_ENABLE(libavcodec-static,
   esac],
   [enable_libavcodec_static=no])
 LIBAVCODEC_EXTRA_LIBS="$LIBAVCODEC_EXTRA_LIBS -lm -lz $PTHREAD_LIBS"
-TC_PKG_CHECK(libavcodec, required, LIBAVCODEC, no, [ffmpeg/avcodec.h],
- avcodec, avcodec_thread_init, FFMpeg, [http://www.ffmpeg.org/])
+TC_PKG_CHECK(libavcodec, required, LIBAVCODEC, no, [ffmpeg/libavcodec/avcodec.h],
+ avcodec, avcodec_thread_init, libavcodec, [http://www.ffmpeg.org/])
 if test x"$have_libavcodec" = x"yes" ; then
   save_CPPFLAGS="$CPPFLAGS"
   CPPFLAGS="$CPPFLAGS $LIBAVCODEC_CFLAGS"
   AC_TRY_RUN([
 #include <stdio.h>
-#include <ffmpeg/avcodec.h>
+#include <libavcodec/avcodec.h>
 int
 main()
 {
@@ -640,8 +640,8 @@ main()
 ],
     [LIBAVCODEC_VERSION="`./conftest$ac_exeext | sed -ne 's,VER=\(.*\),\1,p'`"
       LIBAVCODEC_BUILD="`./conftest$ac_exeext | sed -ne 's,BUILD=\(.*\),\1,p'`"],
-    [TC_PKG_ERROR(libavcodec, ffmpeg/avcodec.h, required, libavcodec, [http://www.ffmpeg.org],
-      [cannot compile ffmpeg/avcodec.h])],
+    [TC_PKG_ERROR(libavcodec, libavcodec/avcodec.h, required, libavcodec, [http://www.ffmpeg.org],
+      [cannot compile libavcodec/avcodec.h])],
     [echo $ac_n "cross compiling; assumed OK... $ac_c"
       LIBAVCODEC_VERSION=""
       LIBAVCODEC_BUILD=""])
@@ -792,7 +792,7 @@ AM_CONDITIONAL(HAVE_IBP, test x"$have_ib
 dnl
 dnl libpostproc
 dnl
-TC_PKG_CHECK(libpostproc, no, LIBPOSTPROC, no, postprocess.h,
+TC_PKG_CHECK(libpostproc, no, LIBPOSTPROC, no, libpostproc/postprocess.h,
   postproc, pp_postprocess, libpostproc, [http://www.ffmpeg.org/])
 TC_PKG_HAVE(libpostproc, LIBPOSTPROC)
 
diff -up transcode-1.0.4/export/aud_aux.c.ffmpeg transcode-1.0.4/export/aud_aux.c
--- transcode-1.0.4/export/aud_aux.c.ffmpeg	2007-08-06 12:56:57.000000000 +0300
+++ transcode-1.0.4/export/aud_aux.c	2008-08-12 19:53:22.000000000 +0300
@@ -31,7 +31,7 @@
 #include <inttypes.h>
 #include <assert.h>
 
-#include <ffmpeg/avcodec.h>
+#include <libavcodec/avcodec.h>
 
 #include "aud_aux.h"
 #include "ac3.h"
diff -up transcode-1.0.4/export/export_ffmpeg.c.ffmpeg transcode-1.0.4/export/export_ffmpeg.c
--- transcode-1.0.4/export/export_ffmpeg.c.ffmpeg	2005-07-12 09:58:36.000000000 +0300
+++ transcode-1.0.4/export/export_ffmpeg.c	2008-08-12 19:53:22.000000000 +0300
@@ -36,7 +36,7 @@
 #include "vid_aux.h"
 // FIXME
 #undef EMULATE_FAST_INT
-#include <ffmpeg/avcodec.h>
+#include <libavcodec/avcodec.h>
 
 #if !defined(INFINITY) && defined(HUGE_VAL)
 #define INFINITY HUGE_VAL
diff -up transcode-1.0.4/export/ffmpeg_cfg.h.ffmpeg transcode-1.0.4/export/ffmpeg_cfg.h
--- transcode-1.0.4/export/ffmpeg_cfg.h.ffmpeg	2004-10-18 05:04:14.000000000 +0300
+++ transcode-1.0.4/export/ffmpeg_cfg.h	2008-08-12 19:53:22.000000000 +0300
@@ -2,7 +2,7 @@
 #define __FFMPEG_CFG_H
 
 #include "libioaux/configs.h"
-#include <ffmpeg/avcodec.h>
+#include <libavcodec/avcodec.h>
 
 //char *lavc_param_vcodec = "mpeg4";
 //extern int lavc_param_vbitrate;
diff -up transcode-1.0.4/filter/filter_resample.c.ffmpeg transcode-1.0.4/filter/filter_resample.c
--- transcode-1.0.4/filter/filter_resample.c.ffmpeg	2005-07-04 10:23:01.000000000 +0300
+++ transcode-1.0.4/filter/filter_resample.c	2008-08-12 19:53:22.000000000 +0300
@@ -30,7 +30,7 @@
 #include "filter.h"
 #include "optstr.h"
 
-#include <ffmpeg/avcodec.h>
+#include <libavcodec/avcodec.h>
 
 static char * resample_buffer = NULL;
 static int bytes_per_sample;
diff -up transcode-1.0.4/filter/pp/filter_pp.c.ffmpeg transcode-1.0.4/filter/pp/filter_pp.c
--- transcode-1.0.4/filter/pp/filter_pp.c.ffmpeg	2005-07-04 10:23:01.000000000 +0300
+++ transcode-1.0.4/filter/pp/filter_pp.c	2008-08-12 19:53:22.000000000 +0300
@@ -35,7 +35,7 @@
 #include <ctype.h>
 #include <inttypes.h>
 
-#include "postprocess.h"
+#include "libpostproc/postprocess.h"
 
 static pp_mode_t *mode[MAX_FILTER];
 static pp_context_t *context[MAX_FILTER];
diff -up transcode-1.0.4/import/decode_lavc.c.ffmpeg transcode-1.0.4/import/decode_lavc.c
--- transcode-1.0.4/import/decode_lavc.c.ffmpeg	2005-07-04 10:23:01.000000000 +0300
+++ transcode-1.0.4/import/decode_lavc.c	2008-08-12 19:53:22.000000000 +0300
@@ -31,7 +31,7 @@
 #ifdef EMULATE_FAST_INT
 #undef EMULATE_FAST_INT
 #endif
-#include <ffmpeg/avcodec.h>
+#include <libavcodec/avcodec.h>
 #include "yuv2rgb.h"
 
 #define READ_BUFFER_SIZE (10*1024*1024)
diff -up transcode-1.0.4/import/import_ffmpeg.c.ffmpeg transcode-1.0.4/import/import_ffmpeg.c
--- transcode-1.0.4/import/import_ffmpeg.c.ffmpeg	2007-08-07 14:30:57.000000000 +0300
+++ transcode-1.0.4/import/import_ffmpeg.c	2008-08-12 19:53:22.000000000 +0300
@@ -38,7 +38,7 @@ static int capability_flag = TC_CAP_YUV 
 
 // FIXME
 #undef EMULATE_FAST_INT
-#include <ffmpeg/avcodec.h>
+#include <libavcodec/avcodec.h>
 
 #include "libvo/yuv2rgb.h"
 #include "avilib/avilib.h"
diff -up transcode-1.0.4/src/transcode.c.ffmpeg transcode-1.0.4/src/transcode.c
--- transcode-1.0.4/src/transcode.c.ffmpeg	2007-07-30 23:01:19.000000000 +0300
+++ transcode-1.0.4/src/transcode.c	2008-08-12 19:53:22.000000000 +0300
@@ -4617,7 +4617,7 @@ int main(int argc, char *argv[]) {
 // unneeded object files from a .a file.
 
 #if 0
-#include <ffmpeg/avcodec.h>
+#include <libavcodec/avcodec.h>
 
 void dummy_avcodec(void) {
   AVCodecContext *ctx = NULL;
