--- transcode-1.1.5/import/v4l/import_v4l.c	2009-02-21 23:01:57.000000000 +0200
+++ transcode-1.1.5/import/v4l/import_v4l.c.fix_v4l_1	2011-06-26 20:09:32.039709334 +0200
@@ -29,7 +29,7 @@
 #include <sys/ioctl.h>
 #include <sys/mman.h>
 
-#include <linux/videodev.h>
+#include <libv4l1-videodev.h>
 
 #define MOD_NAME    "import_v4l.so"
 #define MOD_VERSION "v0.2.0 (2008-10-26)"
--- transcode-1.1.5/import/probe_v4l.c	2011-06-26 20:22:27.898708862 +0200
+++ transcode-1.1.5/import/probe_v4l.c.fix_v4l_1	2011-06-26 20:21:48.066708886 +0200
@@ -31,7 +31,7 @@
 
 #include <sys/ioctl.h>
 
-#include <linux/videodev.h>
+#include <libv4l1-videodev.h>
 
 #if defined(HAVE_LINUX_VIDEODEV2_H) && defined(HAVE_STRUCT_V4L2_BUFFER)
 #define _LINUX_TIME_H

