diff -up transcode-1.0.4/configure.in.libmpeg3 transcode-1.0.4/configure.in
--- transcode-1.0.4/configure.in.libmpeg3	2007-10-12 00:09:23.000000000 +0200
+++ transcode-1.0.4/configure.in	2007-10-12 00:19:53.000000000 +0200
@@ -1065,19 +1065,19 @@ if test x"$have_libmpeg3" = x"yes" ; the
   libmpeg3_inc="no"
   save_CPPFLAGS="$CPPFLAGS"
   CPPFLAGS="$CPPFLAGS $libmpeg3_ii"
+  AC_CHECK_HEADER([libmpeg3.h],
+      [LIBMPEG3_CFLAGS="$libmpeg3_ii"
+        libmpeg3_inc="yes"])
+  if test x"$libmpeg3_inc" = x"no" ; then
   AC_CHECK_HEADER([libmpeg3/libmpeg3.h],
     [LIBMPEG3_CFLAGS="$libmpeg3_ii/libmpeg3"
       libmpeg3_inc="yes"])
+  fi
   if test x"$libmpeg3_inc" = x"no" ; then
     AC_CHECK_HEADER([mpeg3/libmpeg3.h],
       [LIBMPEG3_CFLAGS="$libmpeg3_ii/mpeg3"
         libmpeg3_inc="yes"])
   fi
-  if test x"$libmpeg3_inc" = x"no" ; then
-    AC_CHECK_HEADER([libmpeg3.h],
-      [LIBMPEG3_CFLAGS="$libmpeg3_ii"
-        libmpeg3_inc="yes"])
-  fi
   CPPFLAGS="$save_CPPFLAGS"
   if test x"$libmpeg3_inc" = x"no" ; then
     have_libmpeg3="no"
